server/__init__.py

```diff
@@ -214,6 +217,12 @@ def create_app():
     app.config['BABEL_DEFAULT_LOCALE'] = i18n.DEFAULT_LOCALE
     app.config['BABEL_TRANSLATION_DIRECTORIES'] = 'i18n'
 
+    # load disaster dashboard data frpm GCS
```

Comment:
nit: from.

should it be `get_disaster_dashboard_data` instead of config?

Reply:
done